lib/manager/composer/artifacts.ts

```diff
@@ -134,7 +134,13 @@ export async function updateArtifacts({
         ).trim() + ' --with-dependencies';
     }
     if (config.composerIgnorePlatformReqs) {
-      args += ' --ignore-platform-reqs';
+      if (config.composerIgnorePlatformReqs.length === 0) {
+        args += ' --ignore-platform-reqs';
+      } else {
+        config.composerIgnorePlatformReqs.forEach((req) => {
+          args += ' --ignore-platform-req ' + req;
```

Comment:
```suggestion
          args += ' --ignore-platform-req ' + quote(req);
```

We should add proper quotes to escape improper / malcious values, needs `import { quote } from 'shlex';`

Reply:
Done!